backend/de.metas.manufacturing.webui/src/main/java/de/metas/manufacturing/webui/process/PP_Order_Candidate_SetStartDate.java

```diff
@@ -102,6 +124,14 @@ private ImmutableSet<PPOrderCandidateId> getSelectedPPOrderCandidateIds()
 		}
 	}
 
+	@NonNull
+	private Timestamp convertParamsToTimestamp()
+	{
+		return Timestamp.valueOf(p_Date.toLocalDateTime()
```

Comment:
why not toLocalDate instead of toLocalDateTime.
in this way you make sure you discard also the seconds and millis.

Reply:
changed to LocalDate